.circleci/config.yml

```diff
@@ -36,6 +36,27 @@ parameters:
     default: false
 
 commands:
+  # rn/ios_build depends on a specific cache key, so we need to create a custom step to install dependencies
+  # we could use rn/yarn_install but that's not compatible with yarn 2
```

Comment:
It's explained here, but to be more specific, `rn/ios_build` expects yarn install caches to have been created. There's a command `rn/yarn_install` that creates the expected caches, but the problem with that command is that it doesn't work with yarn 2 (it runs yarn with some unsupported flags).

So until they update the yarn install command in the or, we can use this command, that I copied and modified for our needs from the source code